### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dawnwages**
Keep open until we're ready for the site to go live.

https://[staging-djangonaut-space.azurewebsites.net](https://staging-djangonaut-space.azurewebsites.net/)/

### CODE REVIEW

**Comment by dawnwages** [comment]
@RachellCalhoun no rush on this, but when you get a chance could you check to see if you're getting the same error locally on problem 1? 

Problem 2 is likely a github workflow issue, but if you know if/how/where the problem was introduced that maybe it could be `django-anymail`? I think we're deploying on 3.9 but we may have our local settings on 3.10. I'll check that locally and see if I get the OpenSSL error.

Two (2) issues:

locally when I run `python manage.py runserver --settings=indymeet.settings.production` I get the error that we fixed on the main branch a few days ago. 

```----------------------------------
----------------------------------
PRODUCTION
----------------------------------
----------------------------------
Exception in thread django-main-thread:
Traceback (most recent call last):
  File "C:\Users\dawnwages\AppData\Local\Programs\Python\Python39-32\lib\threading.py", line 950, in _bootstrap_inner
    self.run()
  File "C:\Users\dawnwages\AppData\Local\Programs\Python\Python39-32\lib\threading.py", line 888, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\dawnwages\Projects\djangonaut-space\indymeet\venv\lib\site-packages\django\utils\autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "C:\Users\dawnwages\Projects\djangonaut-space\indymeet\venv\lib\site-packages\django\core\management\commands\runserver.py", line 125, in inner_run
    autoreload.raise_last_exception()
  File "C:\Users\dawnwages\Projects\djangonaut-space\indymeet\venv\lib\site-packages\django\utils\autoreload.py", line 87, in raise_last_exception     
    raise _exception[1]
  File "C:\Users\dawnwages\Projects\djangonaut-space\indymeet\venv\lib\site-packages\django\core\management\__init__.py", line 398, in execute
    autoreload.check_errors(django.setup)()
  File "C:\Users\dawnwages\Projects\djangonaut-space\indymeet\venv\lib\site-packages\django\utils\autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "C:\Users\dawnwages\Projects\djangonaut-space\indymeet\venv\lib\site-packages\django\__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "C:\Users\dawnwages\Projects\djangonaut-space\indymeet\venv\lib\site-packages\django\apps\registry.py", line 91, in populate
    app_config = AppConfig.create(entry)
  File "C:\Users\dawnwages\Projects\djangonaut-space\indymeet\venv\lib\site-packages\django\apps\config.py", line 193, in create
    import_module(entry)
  File "C:\Users\dawnwages\AppData\Local\Programs\Python\Python39-32\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "C:\Users\dawnwages\Projects\djangonaut-space\indymeet\venv\lib\site-packages\captcha\__init__.py", line 18, in <module>
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: ('Setting RECAPTCHA_PRIVATE_KEY is not of type', <class 'str'>)```

2. gh action deploys without any breaking errors, but throws error that does not happen on develop => main: 
```Content in source directory is a Django app
Running collectstatic...
Traceback (most recent call last):
  File "/github/workspace/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
----------------------------------
----------------------------------
PRODUCTION
  File "/github/workspace/pythonenv3.9/lib/python3.9/site-packages/django/core/management/__init__.py", line 446, in execute_from_command_line
    utility.execute()
  File "/github/workspace/pythonenv3.9/lib/python3.9/site-packages/django/core/management/__init__.py", line 420, in execute
    django.setup()
  File "/github/workspace/pythonenv3.9/lib/python3.9/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/github/workspace/pythonenv3.9/lib/python3.9/site-packages/django/apps/registry.py", line 91, in populate
    app_config = AppConfig.create(entry)
  File "/github/workspace/pythonenv3.9/lib/python3.9/site-packages/django/apps/config.py", line 123, in create
    mod = import_module(mod_path)
  File "/tmp/oryx/platforms/python/3.9.15/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/github/workspace/pythonenv3.9/lib/python3.9/site-packages/anymail/apps.py", line 4, in <module>
    from .checks import check_deprecated_settings, check_insecure_settings
  File "/github/workspace/pythonenv3.9/lib/python3.9/site-packages/anymail/checks.py", line 4, in <module>
    from anymail.utils import get_anymail_setting
  File "/github/workspace/pythonenv3.9/lib/python3.9/site-packages/anymail/utils.py", line 13, in <module>
    from requests.structures import CaseInsensitiveDict
  File "/github/workspace/pythonenv3.9/lib/python3.9/site-packages/requests/__init__.py", line 43, in <module>
    import urllib3
----------------------------------
  File "/github/workspace/pythonenv3.9/lib/python3.9/site-packages/urllib3/__init__.py", line 41, in <module>
    raise ImportError(
ImportError: urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with 'OpenSSL 1.1.0l  10 Sep [201](https://github.com/djangonaut-space/wagtail-indymeet/actions/runs/5560930847/jobs/10158257403#step:6:202)9'. See: https://github.com/urllib3/urllib3/issues/[216](https://github.com/djangonaut-space/wagtail-indymeet/actions/runs/5560930847/jobs/10158257403#step:6:217)8
----------------------------------
'collectstatic' exited with exit code 1.
Done in 0 sec(s).
```

**Comment by dawnwages** [comment]
problem 2 solved! pinned urllib3 https://github.com/djangonaut-space/wagtail-indymeet/pull/94/commits/647dcbcaf249a49a8a58241fffad09a859d74a38

**Comment by dawnwages** [comment]
reminder for myself that the error is preventing me from running the first migrate on the database locally. Can do through ssh in the Azure Portal if want to.
```
  raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: ('Setting RECAPTCHA_PRIVATE_KEY is not of type', <class 'str'>)
```

**Comment by dawnwages** [comment]
@tim-schilling You're right it was missing the variable in my `.env` I've added a `template.env` file for future reference.

File: home/static/css/welcome_page.css [hunk 1]
```diff
@@ -20,12 +20,12 @@ body {
 
 a {
     background-color: transparent;
-    color: #71f4f5;
+    color: #5c0287;
```
Comment (tim-schilling) [inline home/static/css/welcome_page.css:23]: @sabderemane These color change aren't compatible with the current landing page. We need to box the landing page in its own stylesheet or find a color that works everywhere (probably unlikely considering the background on the landing page is black and white for the app)
image

**Pull request merged**